contribs/gmf/src/services/querymanager.js

```diff
@@ -128,8 +128,9 @@ gmf.QueryManager.prototype.createSources_ = function(node) {
           }
         }, this);
         layers = childLayerNames.join(',');
-      } else if (node.type === 'WMS') {
-        validateLayerParams = true;
+        if (node.type === 'WMS' && childLayerNames.length == 1) {
+          validateLayerParams = true;
+        }
```

Comment:
I hadn't noticed that nodes with childLayers could have the 'WMS' type.  The logic still works here, so it's fine.